### DISCUSSION TITLE
How are .md files having their layout set?

### DISCUSSION THREAD

**Discussion by c6nabors** [arthelokyo/astrowind#374] [Q&A]
I've been searching through the layout for this theme, and I can't seem to find how .md files are having their theme set, as there is no layout tag inside of their fences.

**Comment by tambuildsthings** [comment]
.md files are part of the post content collection (content/post), which are rendered in the [...blog] / [...page] file, which you can find here: https://github.com/onwidget/astrowind/blob/main/src/pages/%5B...blog%5D/%5B...page%5D.astro
